acceptance/go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	github.com/hashicorp/consul/api v1.10.1-0.20230906155245-56917eb4c968
 	github.com/hashicorp/consul/proto-public v0.5.1
 	github.com/hashicorp/consul/sdk v0.15.0
+	github.com/hashicorp/go-multierror v1.1.1
```

Comment:
weird that this got added? I wonder if we're missing go-mod-tidy's ...  I thought we had CI for this

Reply:
But yeah, I figured the CICD would catch missing go-mod-tidy calls.